File: notes/conditionals.py
# AS 7th Conditionals notes

#homework_done = input("is your homework done: ").strip().capitalize()

#if homework_done == "yes":
    #print("yes, you can go")
#else:
    #print("go do your homework!")

grade = 110

if grade >= 90:
    if grade > 100:
        print(f"Yu cheese it, that isnt possible!")
    else:
       print(f"You have grade {grade}% that is an A!")
elif grade >= 80:
    print(f"you have a {grade}% that is a B!")
elif grade >= 70:
    print(f"you have a {grade}% that is a C!")
else:
    print(f"You have a {grade}% that is not an A :<")

# if is telling the computer if it is true or false
# boolean statement or condition is a true or false (line 5)
# one = is a comand
# two = is a quesion. EX: "are these the same/equal?"
# something indented means its inside of something
# < is greater than
# > is less than
# <= is less than or equal to
# >= is greater than or equal to
# ! means not


# need to write elif when have more than 2 needed outcomes

name = input("What is your name: ")

if name == "Mrs. LaRose": 
    print("You are the teacher!")
elif name == "Tia" or name == "Ashley":
    print("You are the TA")
else:
    print(f"Hello {name}, you are a student!")
    if grade >= 70:
        print("you are passing the class!")
# logical opperators and, or not

File: c_programming/notes/conditionals.c
// AS 7th conditionals notes
#include <stdio.h>
#include <string.h>
/* Logical operators
&& and
|| or
! not
*/

int main(void){
    int grade;
    char name[50];
    printf("what is your grade percent: ");
    scanf("%d", &grade);

    printf("what is your name: ");
    scanf("%s", &name);
    
    //printf("%d", strcmp(name, "Alexa"));
    if(strcmp(name, "Alexa") == 0){
        printf("you dont get a grade!\n");
    }else if(grade >= 90){
        printf("You have an A!\n");
    }else if(grade >= 80){
        printf("You have a B!\n");
    }else if(grade >= 70){
        printf("You have a C!\n");
    }else{
        printf("You are failing. . .\n");
    }

    return 0;
}
//What puts something inside of the “if” statement?
// {} curly brackets
//How are conditions written in C?
// 
//How do we write elif conditions in C?
//
//When do else conditions run?
//